src/Engine/Combat/CombatModifier.java

```diff
@@ -0,0 +1,27 @@
+package Engine.Combat;
+
+import CommandingOfficers.Commander;
+
+public class CombatModifier
```

Comment:
Now that this class has actual functionality, I'd like a description here of what it is, how/where it's used, so a new person can look at the class description and orient themselves.